compiler/rustc_interface/src/passes.rs

```diff
@@ -1011,7 +1011,7 @@ fn run_required_analyses(tcx: TyCtxt<'_>) {
     // Prefetch this to prevent multiple threads from blocking on it later.
     // This is needed since the `hir_id_validator::check_crate` call above is not guaranteed
     // to use `hir_crate`.
```

Comment:
```suggestion
    // to use `hir_crate_items`.
```

Reply:
Also maybe it should access the body of each hir item too?

Reply:
I'm not convinced this is necessary. `hir_crate_items` does this traversal already to collect all the items, including inside bodies.